deepchem/splits/splitters.py

```diff
@@ -260,6 +260,38 @@ def split(self,
     """
     raise NotImplementedError
 
+  def __str__(self) -> str:
```

Comment:
I also added the simple print function. How do you think? 
Many splitter doesn't receive args when instantiating, so `__str__` return just class name.

Reply:
This is great! It's nice to have clean print functions to to make REPL usage friendlier ;)